outer-scripts/kubectl-kubestellar-ensure-location

```diff
@@ -118,11 +118,6 @@ then kubectl ws "${kubectl_flags[@]}" "$imw"
      cwsi=$imw
 fi
 echo "current cwsi is $cwsi"
-if ! kubectl "${kubectl_flags[@]}" get apibinding "edge.kubestellar.io" &> /dev/null; then
```

Comment:
What ensures that the synctargets and locations resources are defined in an IMW (IS)?

Reply:
Yes, we need to ensure this. Let's use https://github.com/kubestellar/kubestellar/issues/1337#issuecomment-1832146503 to track it.